www-apps/jellyfin-bin/jellyfin-bin-10.10.7.ebuild

```diff
@@ -3,33 +3,39 @@
 
 EAPI=8
 
-inherit pax-utils systemd tmpfiles
+inherit pax-utils systemd tmpfiles eapi9-ver
 
 DESCRIPTION="Jellyfin puts you in control of managing and streaming your media"
 HOMEPAGE="https://jellyfin.org/
 	https://github.com/jellyfin/jellyfin/"
-
+MY_PV="${PV//_rc/-rc}"
+MINOR_VER=$(ver_cut 1-2)
+if [[ "${PV}" == *"rc"* ]]; then
```

Comment:
You don't need quotes here

Reply:
My suggestion was to simply `if [[ ${PV} == *rc* ]]; then`

Reply:
But again, that should be a comment for 830e49fcc20ed344a87aac8be220ed06794839dc and modifying jellyfin-bin-10.11.0_rc2.ebuild

Reply:
no problem, and I see the point for a separate commit for the 10.10.7 ebuild update.

Reply:
That's not what I said. :) I said, one could have two commits:
1. 10.10.7: sync style / rc handling from newer ebuild. We will make use of it in the next commit.
2. prevent accidental downgrading (add to both ebuild versions)

Reply:
Ah, then I misunderstood, I did:
  1. Added 10.11.0_rc2 with updated style/warnings (compared to rc1)
  1. sync 10.10.7 ebuild to add downgrade protection

I hope that's fine as well.

Reply:
> Added 10.11.0_rc2 with updated style/warnings (compared to rc1)

Now the commit message for it doesn't mention it adds downgrade protection. :)

Again, *the* (only) reason I mentioned two commits was as a response to *clarify* that the review comment for `if [[ ${PV} == *rc* ]]; then` was inapplicable.